dm/tests/all_mode/run.sh

```diff
@@ -335,6 +335,12 @@ function run() {
 	)
 	export GO_FAILPOINTS="$(join_string \; ${inject_points[@]})"
 
+	# manually create target table with two extra field
+	run_sql_tidb "drop database if exists all_mode;"
+	run_sql_tidb "create database all_mode;"
+	run_sql_tidb "drop table if exists all_mode.no_diff;"
+	run_sql_tidb "create table all_mode.no_diff(id int NOT NULL PRIMARY KEY, dt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP, ts timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP);"
```

Comment:
Could we use `ts timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP on update current_timestamp` to test? This case used to cause a problem

Reply:
It's the `datetime` type that cause the problem, timestamp is actually compitable across different `time_zone`s. BTW, in this test case `DEFAULT CURRENT_TIMESTAMP ` and `DEFAULT CURRENT_TIMESTAMP on update current_timestamp` is almost the same, because SQL from upstream won't contains this field, so the value is always generated by the downstream cluster.